test/present_packages/currently-installed.t

```diff
@@ -0,0 +1,23 @@
+  $ source "$TESTDIR/../helper.sh"
+
+Outputs an indicator next to previously installed packages
```

Comment:
:clap: THANK YOU for writing tests!

I think should be titled with "currently installed", right?